inc/osvr/ClientKit/EyeTracker.h

```diff
@@ -0,0 +1,122 @@
+/** @file
+    @brief Header
+
+    @date 2015
+
+    @author
+    Sensics, Inc.
+    <http://sensics.com/osvr>
+
+*/
+
+// Copyright 2015 Sensics, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// 	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef INCLUDED_EyeTracker_h_GUID_0A75A14B_0F60_4384_A630_0548995F60D3
+#define INCLUDED_EyeTracker_h_GUID_0A75A14B_0F60_4384_A630_0548995F60D3
+
+
+// Internal Includes
+#include <osvr/ClientKit/EyeTracker_decl.h>
+#include <osvr/ClientKit/Interface.h>
+#include <osvr/ClientKit/EyeTrackerC.h>
+#include <osvr/Util/EyeTrackerReportTypesC.h>
+#include <osvr/Util/TimeValue.h>
+#include <osvr/Util/Deletable.h>
+
+// Library/third-party includes
+#include <boost/shared_ptr.hpp>
+#include <boost/noncopyable.hpp>
+
+// Standard includes
+// - none
+
+namespace osvr {
+
+	namespace clientkit {
+		void registerEyeTrackerCallback(Interface &iface, EyeTrackerCallback cb,
```

Comment:
So this file, based on the imaging one, isn't needed (yet) - the reason we needed one for Imaging is because there's complicated data management stuff for imaging. You can probably can it.